src/Prometheus/Storage/APCng.php

```diff
@@ -539,10 +530,9 @@ private function getValues(string $type, array $metaData): array /** @phpstan-ig
         if (isset($metaData['buckets'])) {
             $metaData['buckets'][] = 'sum';
             $labels[] = $metaData['buckets'];
-            $metaData['labelNames'][] = '__histogram_buckets';
```

Comment:
Since I don't use the number of label names anymore, I removed that parameter from buildPermutationTree and removed this line as it seemed unused. I don't see any change in the output for histograms but if this actually used then I can re-add it.